package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mattermost/commonmark",
   "description": "the Mattermost fork of a strongly specified, highly compatible variant of Markdown",
-  "version": "0.30.1-2",
+  "version": "0.30.1-3",
```

Comment:
This is supposed to be on master already, but I wasn't able to push the changes when I made them a couple months ago